code/modules/surgery/surgery_step.dm

```diff
@@ -237,7 +244,7 @@
 
 //Replaces visible_message during operations so only people looking over the surgeon can see them.
 /datum/surgery_step/proc/display_results(mob/user, mob/living/target, self_message, detailed_message, vague_message, target_detailed = FALSE)
-	user.visible_message(detailed_message, self_message, vision_distance = 1, ignored_mobs = target_detailed ? null : target)
+	user.visible_message(detailed_message, self_message, self_message, vision_distance = 1, ignored_mobs = target_detailed ? null : target)
```

Comment:
This is displaying the self message to all blind mobs in view, not just the user, which is improper. You should be using a `to_chat` for the self message.